lib/screens/landing_page/landing_page.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:flutter/material.dart';
+import 'package:promdi_fe/helpers/style.dart';
+import 'package:promdi_fe/screens/login/login.dart';
+import 'package:promdi_fe/screens/signup/signup_page.dart';
+import 'package:promdi_fe/widgets/card.dart';
+
+class LandingPage extends StatefulWidget {
+  const LandingPage({Key? key}) : super(key: key);
+
+  @override
+  _LandingPageState createState() => _LandingPageState();
+}
+
+class _LandingPageState extends State<LandingPage> {
```

Comment:
This design is gonna change as we look to have the design follow the wireframe in the [Google Drive landing page?](https://drive.google.com/file/d/1t7w3J6bvsOtMWhH6O_UkmQg2g5ze_tsr/view?usp=sharing)